### ISSUE TITLE
Biopterin high in urine

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#11618]
**Preferred term label:**
Biopterin high in urine

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Dihydropteridine Reductase Deficiency https://www.ncbi.nlm.nih.gov/books/NBK622830/ "Quantitative assay of urine (or DBS) pterins. Normal (or slightly elevated) neopterin and elevated biopterin"

**Your nano-attribution (ORCID)**
